src/libraries/System.Net.Http/tests/FunctionalTests/MetricsTest.cs

```diff
@@ -584,7 +584,7 @@ public HttpMetricsTest_Http11(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotNodeJS))]
```

Comment:
What about adding a link to issue if we're blocking it?

Reply:
This is a timeout on NodeJS, browser is OK. 
I think this boils down to not perfect implementation of fetch in the npm package and it's not our issue.